src/app/(spaces)/Space.tsx

```
@@ -137,70 +122,38 @@ export default function Space({
-        };
-        Object.entries(keyMap).forEach(([oldKey, newKey]) => {
-          if (oldKey in settings) {
-            settings[newKey] = settings[oldKey];
-            delete settings[oldKey];
-            settingsChanged = true;
-          }
-        });
+      const settings = datum.config?.settings as Record<string, unknown>;
+      if (settings && "fidget Shadow" in settings) {
+        settings.fidgetShadow = settings["fidget Shadow"];
+        delete settings["fidget Shadow"];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)